src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,25 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard';
+import { moviesType } from '../shapeOfCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <MovieCard
+        title={movie.title}
+        description={movie.description}
+        img={movie.imgUrl}
+        imdb={movie.imdbUrl}
+        key={movie.imdbId}
+      />
```

Comment:
https://medium.com/@kevinyckim33/jsx-spread-operator-component-props-meaning-3c9bcadd2493

```suggestion
      <MovieCard {...movie} key={movie.imdbId} />
```